package.json

```diff
@@ -24,7 +24,7 @@
     "release": "release-it"
   },
   "dependencies": {
-    "@ember/render-modifiers": "^1.0.2",
+    "@ember/render-modifiers": "^1.0.2 || ^2.0.0",
```

Comment:
Does updating the lockfile do anything?  Might need it

Reply:
The lockfile of a dependency isn't really used be the consuming app, AFAIK.
It could affect the dummy app, though, but

Reply:
In any case, I updated the lockfile.

Reply:
Mind rebasing with master and we will cut with 4.0?

Reply:
@snewcomer done